### PR TITLE
Add margins to post list for tablets

### PR DESCRIPTION
**Pull request by malinajirka**
Adds start/end margin to Standard (card) layout type on the post list screen.

@SylvesterWilmott I wasn't sure if the margin should be applied even to the FAB.

maxwidth

To test:
Test on phone and tablet
1. My Site
2. Blog posts
3. Click on the "Switch layout type" button in the toolbar
4. Make sure the margin is present only for the standard(card) view + only when the display size is wide enough

Update release notes:

- The improvement is too minor.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt [hunk 1]
```diff
@@ -114,10 +115,20 @@ class PostListFragment : Fragment() {
...
+                                    .getDimensionPixelSize(R.dimen.post_list_content_margin)
+                            it.marginEnd = nonNullActivity.resources
+                                    .getDimensionPixelSize(R.dimen.post_list_content_margin)
+                        }
                     }
                     COMPACT -> {
                         recyclerView?.removeItemDecoration(itemDecorationStandardLayout)
                         recyclerView?.addItemDecoration(itemDecorationCompactLayout)
+                        (recyclerView?.layoutParams as? MarginLayoutParams)?.let {
+                            it.marginStart = 0
```
Comment (shiki) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt]: Will it make sense to have an `R.dimen.post_list_content_margin_compact` even if it's zero?

Comment (malinajirka) [reply WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt]: Good call - fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9798/commits/db959aff4d6adbcdfa807694a36fe25d12985c42

**Comment by malinajirka** [comment]
Thanks for the review @shiki !

>The implementation seems odd to me. I expected the scrollbar to be shown on the right side even if the width is constrained. Just like in web browsers.

I kept it consistent with Stats/Site Creation. However, I just noticed it's not consistent with the Reader. 🤷‍♂Wdyt Sylvester? 

>Unrelated: I noticed some of the posts with no images are laid out like they have featured images.

Good catch. I've decided to fix it here since the fix is super simple https://github.com/wordpress-mobile/WordPress-Android/pull/9798/commits/84252ac2ea30fe91d7deae819fcc52a504a3b11e

**Comment by shiki** [comment]
Recent changes look good and thank you for fixing that unrelated layout issue, @malinajirka. I'll wait for @SylvesterWilmott's response before approving. 🙂

**Comment by SylvesterWilmott** [comment]
Looks great @malinajirka 

Concerning the scrollbar, I think keeping it consistent with the reader is the best way to go, then the scrollbar appears in the same place when you toggle between cards/list.

**Comment by malinajirka** [comment]
Thank you both;)!

>Concerning the scrollbar, I think keeping it consistent with the reader is the best way to go, then the scrollbar appears in the same place when you toggle between cards/list.

Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9798/commits/479794bb1fa45574ee0e32a46b20c17012d7f17d

It's ready for another round @shiki, thanks!! ;)

**Pull request merged**